cmd/minishift/cmd/hostfolder/add.go

```diff
@@ -253,3 +264,21 @@ func addCIFSNonInteractive(manager *hostFolderConfig.Manager, name string) {
 	hostFolder := hostFolderConfig.NewCifsHostFolder(config)
 	manager.Add(hostFolder, !instanceOnly)
 }
+
+// returns the name and full-name for shareType
+func readNameAndTypeInteractive() (string, string) {
+	name := util.ReadInputFromStdin("Name")
+	if name == "" {
+		atexit.ExitWithMessage(1, noName)
+	}
+	shareType := util.ReadInputFromStdin("Type [(s)shfs | (c)ifs]")
```

Comment:
Yes.. it is confusing to me too.. no clue which one is the default. If we add `default: cifs` it will be long and not nice ... but now that you have mentioned about it putting `cifs` first somehow hints it is the default one..